test/Semantics/common.sh

```diff
@@ -10,19 +10,18 @@ function die {
   echo "$(basename $0): $*" >&2
   exit 1
 }
+if [[ $# < 1 ]]; then
+  echo "Usage: $(basename $0) <fortran-source> [<f18-executable>] <temp test dir>"
```

Comment:
The [ ] syntax in the usage string is meant to indicate an optional argument. In the new interface, the F18 executable is non-optional so these should be removed. I think all the arguments are non-optional in the new interface.